staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -160,8 +160,8 @@ type delayingType[T comparable] struct {
 	// waitingForAddCh is a buffered channel that feeds waitingForAdd
 	waitingForAddCh chan *waitFor
 
-	// metrics counts the number of retries
-	metrics retryMetrics
+	// metrics for the delay queue
+	metrics delayQueueMetrics
```

Comment:
why the rename?

Reply:
`retryMetrics` doesn't seem appropriate anymore for the struct as it stores metrics for delay queue length besides retries as well.